test/functionalTest/cases/1005_GET_v2_entities_string/1005_GET_v2_entities_string.test

```diff
@@ -0,0 +1,137 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Actually deferred to tech debt issue: https://github.com/telefonicaid/fiware-orion/issues/1010